mne_sandbox/decoding/_time_frequency_csp.py

```diff
@@ -0,0 +1,255 @@
+# -*- coding: utf-8 -*-
+# Authors: Laura Gwilliams <laura.gwilliams@nyu.edu>
+#          Jean-Remi King <jeanremi.king@gmail.com>
+#          Alex Barachant <alexandre.barachant@gmail.com>
+#          Alexandre Gramfort <alexandre.gramfort@telecom-paristech.fr>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+from mne.filter import filter_data
+
+
+class TimeFrequencyCSP(object):
+    u"""Decode MEG data in time-frequency space using a rolling covariance
+    matrix.
+
+    This object can be used as a supervised decomposition to estimate
+    spatial filters for feature extraction in a 2 class decoding problem.
+
+    Parameters
+    ----------
+    tmin : float
+        Time in seconds. Lower bound of the time over which to estimate
+        signals.
+    tmax : float
+        Times in seconds. Upper bound of the time over which to estimate
+        signals.
+    freqs : array of floats
+        Frequency values over which to compute time-frequency decomposition.
```

Comment:
Can you elaborate a bit on how exactly to pass the frequency ranges? Maybe give an example here?